packages/web/src/pages/upload-page/fields/ToggleRowField.tsx

```diff
@@ -0,0 +1,35 @@
+import { PropsWithChildren } from 'react'
+
+import { Switch } from '@audius/stems'
+import cn from 'classnames'
+import { useField } from 'formik'
+
+import styles from './ToggleRowField.module.css'
+
+type ToggleFieldProps = PropsWithChildren & {
+  name: string
+  header: string
+  description: string
+}
+
+export const ToggleRowField = (props: ToggleFieldProps) => {
+  const { name, header, description, children } = props
+  const [{ value }, , { setValue }] = useField<boolean>(name)
```

Comment:
can we use the onChange from the useField? personally i like using that over setValue whenever possible

Reply:
yea I think I got into a habit of using setValue cause it wasn't working with the old fields, but this one should work with onChange